pkg/asset/installconfig/aws/validation.go

```diff
@@ -17,64 +19,66 @@ import (
 )
 
 // Validate executes platform-specific validation.
-func Validate(ctx context.Context, meta *Metadata, config *types.InstallConfig) error {
```

Comment:
any reason why we remove the ctx ?

Reply:
I've removed the mock framework from AWS.